package.json

```diff
@@ -36,7 +36,7 @@
     "lint": "./node_modules/.bin/standard",
     "lint-sass": "./node_modules/.bin/sass-lint -v",
     "test": "npm run snyk-protect && rm -rf ./pacts && ./node_modules/mocha/bin/mocha '!(node_modules)/**/*+(.|_)+(test|tests)'.js",
-    "pact-stub": "docker run -t -p 8000:8000 -v \"$(pwd)/pacts/:/app/pacts\" pactfoundation/pact-stub-server -p 8000 -d pacts",
+    "mb": "docker run -t -p 2525:2525 -p 8000:8000 -d andyrbell/mountebank",
```

Comment:
We can install and run mountebank using npm, but this would be an extra step for people the first time they run to install it globally, and I'm not sure about including as a dependency as the project doesn't actually use it. I guess we'd have to run it in docker on CI too so maybe it's best to run it in the same way locally?
I've included the most used image from DockerHub here as there isn't an official one, but what is our usual approach for docker images, would we prefer to build our own?

Reply:
Made this change